src/net/p2p.rs

```diff
@@ -3010,6 +3063,29 @@ impl PeerNetwork {
                                 "{:?}: Finished full inventory state-machine pass ({})",
                                 &self.local_peer, self.num_inv_sync_passes
                             );
+
+                            // hint to the downloader to start scanning at the sortition
+                            // height we just synchronized
+                            let start_download_sortition =
+                                if let Some(ref inv_state) = self.inv_state {
+                                    debug!(
+                                    "{:?}: Begin downloader synchronization at sortition height {}",
+                                    &self.local_peer, inv_state.block_sortition_start
+                                );
+                                    inv_state.block_sortition_start
+                                } else {
+                                    // really unreachable, but why tempt fate?
```

Comment:
Should we emit an error log in this case?

Reply:
Done